documentation/modules/configuring/con-config-mirrormaker2-tasks-max.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-config-mirrormaker2.adoc
+
+[id='con-mirrormaker-tasks-max-{context}']
+= Specifying a maximum number of tasks
+
+[role="_abstract"]
+Connectors create the tasks that are responsible for moving data in and out of Kafka.
+Each connector comprises one or more tasks that are distributed across a group of worker pods that run the tasks.
+
+Tasks run in parallel.
+If there are more tasks than workers, workers handle multiple tasks.
```

Comment:
I think the text is good. But I think it is worth mentioning that a single task will run only within one worker pod. (i.e. having replicas=10 but keeping the tasksMax=1 is useless). I think it kind of follows from the text, but might not hurt to be explicit about this I think

Reply:
Changed to:

> Tasks run in parallel.
> Workers are assigned one or more tasks.
> A single task is handled by one worker pod, so you don't need more worker pods than tasks.
> If there are more tasks than workers, workers handle multiple tasks.